src/components/App/Alerts/CreateAlert.vue

```diff
@@ -237,6 +244,8 @@ export default class CreateAlert extends Mixins(
       this.selectAsset(this.xor);
     }
 
+    this.floatInput?.$children?.[0]?.focus?.(); // price input autofocus
```

Comment:
`this.floatInput.focus()` is not working in this case?

Reply:
no